notebooks/image_models/solutions/vertex_tuning_image_classification.ipynb

```diff
@@ -0,0 +1,1225 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    train_ds = create_dataset(
</pre><p>If it is just for validation, different variable name may be better (like <code>temp_ds</code>), as it can be confusing to use the same variable that is also used in train.py.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/225/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>Done.</p>